### PR TITLE
Removing backticks and integrating feedback from Artem

### PR DESCRIPTION
**Pull request by Buzzardo**
I removed the backticks that were making links not render. I fixed a few typoes, too.

Then Artem caught some errors that I had made. I fixed them. 

Thanks, Artem. :)

### CODE REVIEW

**Comment by artembilan** [comment]
@Buzzardo ,

since #2427 has been just merged (thanks, @garyrussell !), please, consider to rebase your PR to the latest `master`.

**Comment by garyrussell** [comment]
FYI, we have a lot of PDF overflows since we merged Jay's last PR:

```
Line 1 of a paragraph overflows the available area by 22824mpt. (fo:block, location: 2116/1075)
Line 1 of a paragraph overflows the available area by 39624mpt. (fo:block, location: 2117/1075)
Line 3 of a paragraph overflows the available area by 1824mpt. (fo:block, location: 2117/2156)
Line 1 of a paragraph overflows the available area by 39624mpt. (fo:block, location: 2118/1146)
Line 3 of a paragraph overflows the available area by 1824mpt. (fo:block, location: 2118/2096)
Line 1 of a paragraph overflows the available area by 46496mpt. (fo:block, location: 2127/3572)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2127/6208)
Line 1 of a paragraph overflows the available area by 42296mpt. (fo:block, location: 2127/9065)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2127/11723)
Line 1 of a paragraph overflows the available area by 21296mpt. (fo:block, location: 2128/1285)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2128/4047)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2129/1125)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2130/1177)
Line 1 of a paragraph overflows the available area by 14066mpt. (fo:block, location: 2130/2279)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2130/3879)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2130/4904)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 2140/3572)
Line 1 of a paragraph overflows the available area by 35066mpt. (fo:block, location: 2140/4674)
Line 1 of a paragraph overflows the available area by 38096mpt. (fo:block, location: 2140/6316)
Line 1 of a paragraph overflows the available area by 30866mpt. (fo:block, location: 2140/7330)
Line 2 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 9819/11)
Line 5 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 9820/96)
Line 1 of a paragraph overflows the available area by 45925mpt. (fo:block, location: 16201/629)
Line 1 of a paragraph overflows the available area by 3900mpt. (fo:block, location: 17765/215)
Line 8 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 20348/96)
Line 2 of a paragraph overflows the available area by 20328mpt. (fo:block, location: 22643/2360)
Line 2 of a paragraph overflows the available area by 38328mpt. (fo:block, location: 22643/4741)
Line 2 of a paragraph overflows the available area by 8328mpt. (fo:block, location: 22644/4950)
Line 1 of a paragraph overflows the available area by 436mpt. (fo:block, location: 22669/3491)
Line 1 of a paragraph overflows the available area by 6436mpt. (fo:block, location: 22681/639)
Line 1 of a paragraph overflows the available area by 436mpt. (fo:block, location: 22684/4450)
Line 1 of a paragraph overflows the available area by 436mpt. (fo:block, location: 22693/4451)
Line 1 of a paragraph overflows the available area by 436mpt. (fo:block, location: 22703/3069)
Line 1 of a paragraph overflows the available area by 436mpt. (fo:block, location: 22714/3087)
Line 1 of a paragraph overflows the available area by 21300mpt. (fo:block, location: 22731/696)
Line 1 of a paragraph overflows the available area by 15300mpt. (fo:block, location: 22731/1238)
Line 1 of a paragraph overflows the available area by more than 50 points. (fo:block, location: 22732/975)
```

I have a patch to the dockbook plugin that helps me track these down (by keeping the `index.fo` file on disk).

2 questions:

1. Should I track these down now?
2. Will the new toolchain identify these overflows?

(Both are moot if we will no longer create PDFs).

**Comment by garyrussell** [comment]
Ah - I looked at the first couple and they were caused changes like this:

https://github.com/spring-projects/spring-integration/commit/f3bf2a68d6ac2ee519ea187b062a2a7640309970#diff-965cbde06fa2035c531d4b5f7f2c1a8bL101

Notice I previously had the constant split over two lines - this was so they would fit in the table column. You can see the results by going to page 54 here: https://docs.spring.io/spring-integration/docs/5.1.0.BUILD-SNAPSHOT/reference/pdf/spring-integration-reference.pdf - see how the header names bleed into the next column.

**Comment by garyrussell** [comment]
@Buzzardo , please don't take any action on this; since I have captured the index.fo file it will be easier for me to locate them all rather than you trying to find them all by going through the commits.

**Review by artembilan — CHANGES_REQUESTED** [review]
OK! I only caught one concern.
Otherwise it is good to go.

Thank you!

File: src/reference/asciidoc/jdbc.adoc [hunk 1]
```diff
@@ -831,6 +831,11 @@ Since Spring Integration 3.0.
 Optional.
 ====
 
+=======
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply src/reference/asciidoc/jdbc.adoc]: I think that line of = signs is extraneous. It may be left over from a conflict.

**Comment by garyrussell** [comment]
https://github.com/spring-projects/spring-integration/pull/2514

**Review by artembilan — APPROVED** [review]
Merging with possible minor fixing after rebasing...

**Pull request closed without merging**